solutions/standard/variables.tf

```diff
@@ -0,0 +1,141 @@
+  default     = "us-south"
+}
+
+variable "repos_git_token_secret_id" {
+  type        = string
+  description = "The secretID where the value for the GitHub token is stored in the existing Secrets Manager instance."
+  default     = null
+}
+
+variable "repos_git_token" {
+  type        = string
+  description = "The GitHub token to read from the application and configuration repos. If var.repos_git_token_existing_secrets_manager_id is not null, var.repos_git_token is not used."
+  default     = null
+  sensitive   = true
+}
+
+###################################################
+# MQ S2S policy definition
+###################################################
+
+variable "mq_s2s_policy_enable" {
+  type        = bool
+  description = "Flag to enable creation of the Service to Service policy to enable the Enterprise Application Service instance to reach MQ instance. Default to false"
+  default     = false
+}
+
+variable "mq_s2s_policy_roles" {
+  type        = list(string)
+  description = "List of roles to be configured for the Service to Service policy. Default to Viewer role."
+  default     = ["Viewer"]
+  validation {
+    condition = alltrue([
+      for role in var.mq_s2s_policy_roles : contains(["Administrator", "Viewer", "Operator", "Editor"], role)
+    ])
+    error_message = "The values of var.mq_s2s_policy_roles must be one or more of \"Administrator\", \"Viewer\", \"Operator\", \"Editor\""
+  }
+}
+
+variable "mq_s2s_policy_source_account_id" {
+  type        = string
+  description = "Source accountID for the Service to Service policy from the Enterprise Application Service instance to MQ instance. If mq_s2s_policy_enable is true and this is left to null the accountID of the API key configured for the provider is used"
+  default     = null
+}
+
+variable "mq_s2s_policy_target_account_id" {
+  type        = string
+  description = "Target accountID for the Service to Service policy from the Enterprise Application Service instance to MQ instance. If mq_s2s_policy_enable is true and this is left to null the accountID of the API key configured for the provider is used"
+  default     = null
+}
+
+variable "mq_s2s_policy_limit_source_resource_flag" {
+  type        = bool
+  description = "Flag to limit the source of the Service to Service policy to the created Enterprise Application Service resource instance ID. If false the Service to Service policy source scope is not limited to the resource instance ID. Configuring scope on the resource instance ID and on the resource group ID is mutually exclusive."
+  default     = true
+}
+
+variable "mq_s2s_policy_limit_source_resource_group_flag" {
+  type        = bool
+  description = "Flag to limit the source of the Service to Service policy to the created Resource Group ID. If false the Service to Service policy source scope is not limited to the resource group ID. Configuring scope on the resource instance ID and on the resource group ID is mutually exclusive."
+  default     = false
+}
```

Comment:
why is `mq_s2s_policy_limit_source_resource_flag` set to true, and `mq_s2s_policy_limit_source_resource_group_flag` set to false? Its not consistent. I'm wondering why we even need these booleans at all. Can we not just determine if user passes an ID, it means they want to scope to resource group ID? If its `null`, they dont?

Reply:
The flags are used to allow the user to setup the S2S auth policy with different sources and targets attributes. 
Considering the adoption of this DA (along with other DAs) probably makes sense to reduce its complexity as not needed anymore since when we have been initially asked for such capabilities.
I will reduce the options to allow the S2S from resource to resource, while the other options (S2S with account or resource-group scope) will be implemented externally from the DA itself

Reply:
reviewed the input vars and their validation